modules/apps/headless/headless-builder/headless-builder-test/src/testIntegration/java/com/liferay/headless/builder/resource/test/HeadlessBuilderResourceTest.java

```diff
@@ -2020,7 +2219,196 @@ private void _addAPIApplication(
 			apiEndpointExternalReferenceCode, apiSchemaExternalReferenceCode);
 	}
 
+	private void _addAPIApplicationWithGetAndPostEndpoint(
```

Comment:
So @4lejandrito and I discussed about it the other day and we want to do a refactor to the whole test class. The thing is that it would take a long time and given the prioritization of SVs we thought that it was better to relegate to a future task.

Reply:
We will do this separately. See https://github.com/liferay-headless/liferay-portal/pull/1718.